public/docs/_examples/cb-dynamic-form/ts/app/dynamic-form-question.component.html

```diff
@@ -1,13 +1,13 @@
 <!-- #docregion -->
 <div [ngFormModel]="form">
-  <div class="formHeading">{{question.label}}</div>
+  <label attr.for={{question.key}}>{{question.label}}</label>
```

Comment:
Couldn't this be done like:

```
<label [attr.foo]="question.key"></label>
```

I think that way is more angularish2.

On the other hand, you are missing quotes in there, right?